### PR TITLE
Display not found pages for wrong routes, and unfound cats and foster homes 

### PR DESCRIPTION
**Pull request by mwraich**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,21 +1,56 @@
...
+  render() {
+    return (
+      <div className="App">
+        {this.state.loaded && (
+        <Switch>
+          <Route exact path="/" component={SearchView} />
+          <Route exact path="/match" component={MatchView} />
+          <Route exact path="/catProfile/:catId/:name" component={CatProfileView} />
+          <Route exact path="/fosterProfile/:fosterId/:name" component={FosterProfileView} />
+          <Route exact path="/fosterProfile/:fosterId/:name" component={FosterProfileView} />
```
Comment (auredufour) [inline src/App.jsx]: One too many!

File: src/App.jsx [hunk 2]
```diff
@@ -1,21 +1,56 @@
...
+      </div>
+    );
+  }
 }
 
-export default App;
+const mapStateToProps = state => ({
+  cats: state.cats.cats,
+  fosterHomes: state.fosterHomes.fosterHomes,
+  catID: 1
```
Comment (auredufour) [inline src/App.jsx]: Should we remove it now?

File: src/App.jsx (hunk 1 above)
Comment (mwraich) [reply src/App.jsx]: 310a3c347a876d421caa69a80c896d706503c1df

File: src/App.jsx (hunk 2 above)
Comment (mwraich) [reply src/App.jsx]: 310a3c347a876d421caa69a80c896d706503c1df

**Pull request merged**